code/game/objects/items/crayons.dm

```diff
@@ -427,7 +427,7 @@
 				if(C.reagents)
 					C.reagents.add_reagent("welding_fuel", 5)
 					C.reagents.add_reagent("ethanol", 5)
-					C.reagents.reaction(C, VAPOR, 10)
+					C.reagents.reaction(C, VAPOR)
```

Comment:
C.reagents.reaction(C) ...........
This is making the reagents inside you react with yourself.........